VERSION

```diff
@@ -1 +1 @@
-3.11.0
+3.12.0
```

Comment:
The bump is for testing right?

Reply:
Yep, the whole second commit is only for testing purposes.